src/main/java/com/iridium/iridiumskyblock/commands/SetHomeCommand.java

```diff
@@ -0,0 +1,64 @@
+package com.iridium.iridiumskyblock.commands;
+
+import com.iridium.iridiumskyblock.IridiumSkyblock;
+import com.iridium.iridiumskyblock.api.IridiumSkyblockAPI;
+import com.iridium.iridiumskyblock.database.User;
+import com.iridium.iridiumskyblock.utils.StringUtils;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Command which sets a User's Island Home'
+ */
+public class SetHomeCommand extends Command {
+
+    /**
+     * The default constructor.
+     */
+    public SetHomeCommand() {
+        super(Collections.singletonList("sethome"), "Set the island home", "", true);
+    }
+
+    /**
+     * Executes the command for the specified {@link CommandSender} with the provided arguments.
+     * Not called when the command execution was invalid (no permission, no player or command disabled).
+     * Sets island home of the user.
+     *
+     * @param sender The CommandSender which executes this command
+     * @param args   The arguments used with this command. They contain the sub-command
+     */
+    @Override
+    public void execute(CommandSender sender, String[] args) {
+        Player player = (Player) sender;
+        User user = IridiumSkyblockAPI.getInstance().getUser(player);
+        if (user.getIsland() != null) {
+            // TODO: There should be a is safe method
+            if (true) {
```

Comment:
create a method in  IslandManager called isLocationSafe that takes  in an  island and a location
(just return true for now, ill implement it tomorrow)

Reply:
okay i'll do